apps/system/flash_test/flash_test.c

```diff
@@ -143,19 +633,68 @@ int flash_test_main(int argc, char *argv[])
 #endif
 {
 	off_t offset;
-	int read_size;
+	int size;
+	flash_t flash_type;
 
-	if (argc == 4 && !strncmp(argv[1], READ_CMD, sizeof(READ_CMD) + 1)) {
+	if ((argc == 4) && (!strncmp(argv[1], BCDEV_CMD, sizeof(BCDEV_CMD) + 1))) {
+		flash_type = (flash_t) atoi(argv[3]);
```

Comment:
Let's remove a space after typecasting. There are several lines for the same.

Reply:
updated